src/structures/member.ts

```diff
@@ -32,7 +34,7 @@ export class Member extends Base {
   protected readFromData (data: MemberPayload): void {
     super.readFromData(data.user)
     this.nick = data.nick ?? this.nick
-    this.roles = data.roles ?? this.roles
+    this.roleIDs = data.roles ?? this.roles
```

Comment:
this.roleIDs wants `string[]` but this.roles is `Role[]`

Reply:
Fixed it